File: app/src/main/java/com/example/fpmobdev/Budget.java
package com.example.fpmobdev;

public class Budget {
    public String category;
    public double amount;
    public double remaining;
    public double alertThreshold;

    public Budget() {}

    public Budget(String category, double amount, double remaining, double alertThreshold) {
        this.category = category;
        this.amount = amount;
        this.remaining = remaining;
        this.alertThreshold = alertThreshold;
    }
}

File: app/src/main/java/com/example/coincraft/Budget.kt
package com.example.coincraft

class Budget(
    var category: String = "",
    var amount: Double = 0.0,
    var remaining: Double = 0.0,
    var alertThreshold: Double = 0.0
) {
    constructor() : this("", 0.0, 0.0, 0.0) // Default constructor for Firebase
}
